### PR TITLE
Translate the newly added text into Ukrainian

### PR DESCRIPTION
**Pull request by megamanyoutuber**
The title explains everything that this pull request is supposed to add.

### CODE REVIEW

File: Profile Folder/chrome/locale/uk/dtd/gsettings.dtd [hunk 1]
```diff
@@ -66,7 +66,7 @@
 <!ENTITY gsettings.toolbar						"Панель Інструментів">
 <!ENTITY gsettings.needsRestart					"Потребує перезапуск">
 <!ENTITY gsettings.tabManager					"Менеджер вкладок">
-<!ENTITY gsettings.tabManagerDescription		'[!!! Éñáβℓèƨ ƭλè "ářřôω βúƭƭôñ" ωïƭλ ƭλè ℓïƨƭ ôƒ ôƥèñ ƭáβƨ ïñ ƭλè ƭáββář !!!]'>
+<!ENTITY gsettings.tabManagerDescription		'Вмикає "кнопку стрілочку" зі списком відкритих вкладок у панелі вкладок'>
```
Comment (angelbruni) [inline Profile Folder/chrome/locale/uk/dtd/gsettings.dtd:69]: Would it be possible to find a way to not use "Enable" in `gsettings.tabManagerDescription` and `gsettings.enableForceColorizeAddressBar`? The toggle already implies that it's for "turning on".

**Review by angelbruni — CHANGES_REQUESTED** [review]

File: Profile Folder/chrome/locale/uk/dtd/gsettings.dtd [hunk 2]
```diff
@@ -82,8 +82,8 @@
 <!ENTITY gsettings.disabled									"Ввімкнено">
 <!ENTITY gsettings.chrThemesMustAero						"Примусити усі теми Chromium використовувати системні межі">
 <!ENTITY gsettings.chrThemesMustAeroDisclaimer				"Потребує увімкнення системних меж у підтриманній темі Chromium">
-<!ENTITY gsettings.enableForceColorizeAddressBar			"[!!! Éñáβℓè ÛR£ ßář ƭλè₥ïñϱ ƒôř ïñçô₥ƥáƭïβℓè δèƨïϱñƨ !!!]">
-<!ENTITY gsettings.enableForceColorizeAddressBarDisclaimer	"[!!! Rè9úïřèƨ Gèçƙïú₥ Fïřèƒôж Tλè₥è ₥ôδè ôř á çô₥ƥáƭïβℓè Çλřô₥ïú₥ Tλè₥è !!!]">
+<!ENTITY gsettings.enableForceColorizeAddressBar			"Вмикає темування панелей URE для несумісних дизайнів">
```
Comment (angelbruni) [inline Profile Folder/chrome/locale/uk/dtd/gsettings.dtd:85]: Would it be possible to find a way to not use "Enable" in `gsettings.enableForceColorizeAddressBar`? The toggle already implies that it's for "turning on".

**Review by angelbruni — CHANGES_REQUESTED** [review]

File: Profile Folder/chrome/locale/uk/dtd/gsettings.dtd (hunk 2 above)
Comment (megamanyoutuber) [reply Profile Folder/chrome/locale/uk/dtd/gsettings.dtd:85]: Don't think so as far as I know.

Comment (angelbruni) [reply Profile Folder/chrome/locale/uk/dtd/gsettings.dtd:85]: > Don't think so as far as I know.

I don't need any form of "Enable"/"Turn on", simply "Colorize Address Bar according to custom theme" is enough, it's just the name of the setting, nothing else.

Comment (megamanyoutuber) [reply Profile Folder/chrome/locale/uk/dtd/gsettings.dtd:85]: Ah, that would be something like "Колюрувати адресну строку по темі". I can make that into a commit on my own repository.

Comment (angelbruni) [reply Profile Folder/chrome/locale/uk/dtd/gsettings.dtd:85]: Thank you so much for your help once again, I really appreciate it :)

**Review by angelbruni — APPROVED** [review]

**Comment by angelbruni** [comment]
Thanks!

**Pull request merged**